index.html

```diff
@@ -2327,39 +2327,39 @@ <h4>
         <span its-locale-filter-list="zh-hans" lang="zh-hans">纵横对齐下的中西文混排处理</span>
         <span its-locale-filter-list="zh-hant" lang="zh-hant">縱橫對齊下的中西文混排處理</span>
       </h4>
 
-      <p its-locale-filter-list="en" lang="en">Due to the fact that each Han character is of the same width, not only should characters at the start and end of a line be aligned but it is also a requirement for characters within blocks of Han text to be aligned both vertically and horizontally, whether in vertical or horizontal writing mode.  When Western text or European numerals are present, this principle is harder to achieve. Possible approaches are listed below:</p>
-      <p its-locale-filter-list="zh-hans" lang="zh-hans">由于汉字各字等宽的性质，无论在直排或横排的情况下，除了行齐首尾对齐外，也要求各行间的各个汉字能够纵横对齐。若遇西文或阿拉伯数字采用比例字体，难以满足这项原则时，处理方式如下：</p>
-      <p its-locale-filter-list="zh-hant" lang="zh-hant">由於漢字各字等寬的性質，無論在直排或橫排的情況下，除了行齊頭尾對齊外，亦求各行間的各個漢字能夠縱橫對齊。若遇西文或阿拉伯數字採用比例字體，便難以滿足這項原則，處理方式如下：</p>
+      <p its-locale-filter-list="en" lang="en">Due to the fact that each Han character is of the same width, not only should characters at the start and end of a line be aligned but it is also a requirement for characters within blocks of Han text to be aligned both vertically and horizontally, whether in vertical or horizontal writing mode. When European alphas or European numerals are present, this principle is harder to achieve. Possible approaches are listed below:</p>
+      <p its-locale-filter-list="zh-hans" lang="zh-hans">由于汉字各字等宽的性质，无论在直排或横排的情况下，除了行齐首尾对齐外，亦求各行间的各个汉字能够纵横对齐。若遇西文字母或阿拉伯数字采用比例字体，难以满足这项原则时，处理方式如下：</p>
+      <p its-locale-filter-list="zh-hant" lang="zh-hant">由於漢字各字等寬的性質，無論在直排或橫排的情況下，除了行齊頭尾對齊外，亦求各行間的各個漢字能夠縱橫對齊。若遇西文字母或阿拉伯數字採用比例字體，便難以滿足這項原則，處理方式如下：</p>
       <ol>
         <li id="id119">
-          <p its-locale-filter-list="en" lang="en" class="checkme">Instead of 1/4-em spacing between Han and Western letters, it is possible to use flexible spacing of up to 1/2&nbsp;em. This brings the space occupied by Western characters to a multiple of the width of a Han character. In this way, both the Han character before and after the Western language span snaps to the grid lines.</p>
-          <p its-locale-filter-list="zh-hans" lang="zh-hans" class="checkme">汉字与西文字母间不使用四分之一汉字宽的字距，而是加入大于零、小于等于二分之一汉字宽的弹性空白，使西文所占的空间为汉字的整数倍，确保西文前后的汉字都能纵横对齐。</p>
-          <p its-locale-filter-list="zh-hant" lang="zh-hant">漢字與西文字母間不使用四分之一漢字寬之字距，而是加入大於零、小於等於二分之一漢字寬的彈性空白，使西文所佔之空間為漢字的整數倍，確保西文前後之漢字都能縱橫對齊。</p>
+          <p its-locale-filter-list="en" lang="en" class="checkme">Instead of 1/4-em spacing between a Han character and a letter of European alphas or European numerals, it is possible to use flexible spacing of up to 1/2&nbsp;em. This brings the space occupied by European text to a multiple of the width of a Han character. In this way, both the Han characters before and after a European text span snap to the grid lines.</p>
```

Comment:
'a letter of European alphas or European numerals' is easier to say as 'European alphanumerics'.

Reply:
Thanks. Fixed by [c05fa53](https://github.com/w3c/clreq/pull/492/commits/c05fa5353b8c368f7b21dfa8c7f4b9e584006138).